### ISSUE TITLE
Question: Failed to connect to Metro Flipper did find a running Metro instance

### ISSUE THREAD

**Issue by Efigenio-fidelis** [facebook/flipper#2870]
Failed to connect to Metro
Flipper did find a running Metro instance, but couldn't connect to the logs. Probably your React Native version is too old to support Flipper. Cause: Failed to get a connection to ws://localhost:8081/events in a timely fashion

![Screenshot at Sep 16 09-27-16](https://user-images.githubusercontent.com/82679917/133612008-6a6133a3-2bad-40cd-9aa4-9e8ad11ba622.png)

**Comment by mweststrate** [comment]
And you are running Metro, a recent version? Any errors in the Metro logs?

**Comment by Efigenio-fidelis** [comment]
> And you are running Metro, a recent version? Any errors in the Metro logs?

Yes, I'm running a recent version, and there's no errors shown in the metro logs

**Comment by mweststrate** [comment]
If you run the ReactNativeFlipperExample from our repo, does the same thing happen?

**Comment by mweststrate** [comment]
And, can you list the output of `http://localhost:8081/json`?

**Comment by nicode-io** [comment]
Same error for me today, very difficult to reproduce as it happens when nothing special was done in project ...

**Comment by devben** [comment]
Downgrading flipper fixed this for me. See https://github.com/facebook/flipper/issues/3189
downgrade here: https://github.com/facebook/flipper/releases
